### DISCUSSION TITLE
Curious how something works xD, And request to be an early tester/contributer

### DISCUSSION THREAD

**Discussion by franzsilva** [dotnet/maui#247] [General]
Hi .NET Team. 

Im curious to know how the deploy to ios will work on windows, as Scott Hunter showed on the demo. Will it be something with the power of azure to compile the app and use a cloud service to send the app the the ios Device?, 

I would love to contribute to this project as much as i can, I have a great love for Microsoft Tech, specially what has been done with Open Source .NET. I would love to give this a test drive and report any findings. And also spread the word with articles and progress. 

Well i guess that is all. 

Thanks!

**Comment by AmSmart** [comment]
For the iOS deploy on windows, I believe it's closely bound to Hot Restart which is already available on Xamarin.Forms right now. On the contribution side of things, I believe @PureWeen mentioned that the repo is still in flux for now, so we just have to wait for a couple of weeks till it's stable.
